### ISSUE TITLE
Spawn `electron .` automatically in the "dev" script?

### ISSUE THREAD

**Issue by Terrabits** [PhillMcKraken/enhanced-electron-react-boilerplate#3]
Hi,

Is there a reason why you separate the dev commands into two commands to be run in separate consoles?

Another approach would be to spawn a separate electron process in `webpack.dev.config.js`. Borrowing heavily from `electron-react-boilerplate`, I was able to do the following:

In `webpack.dev.config.js`, I required `child_process.spawn`:

```javascript
const { spawn } = require('child_process');
```

In the `devServer` section, I added:

```javascript
setup() {
      spawn(
        'npm',
        ['run', 'testDev'],
        { shell: true, env: process.env, stdio: 'inherit' }
      )
      .on('close', code => process.exit(0))
      .on('error', spawnError => console.error(spawnError));
}
```

See:
https://github.com/chentsulin/electron-react-boilerplate/blob/master/webpack.config.renderer.dev.js

Is there a disadvantage to doing this? If it does make sense to do this, can I submit a pull request?

This project (and the `basic-`) version are great BTW. A much needed update to something like `electron-react-boilerplate`. THANKS!

**Comment by pbarbiero** [comment]
Thats actually something I had been meaning to look into for a while now. I have it the way it is now as it was the least effort and works well, but it is slightly clunky I agree.

Sure! Make a pull, I would love to integrate that.

**Comment by Terrabits** [comment]
Great! Would you prefer that I keep both a command to start the dev server (as `dev` does now) and a command to start electron (`testDev`)? Or should I combine the two into `dev` or do something else entirely?

**Comment by pbarbiero** [comment]
Merged #4
